manifests/check.pp

```diff
@@ -143,8 +161,8 @@
     publish             => $publish,
     dependencies        => $dependencies,
     custom              => $custom,
-    require             => File['/etc/sensu/conf.d/checks'],
-    notify              => $::sensu::check_notify,
+    require             => File["${conf_dir}/checks"],
+    notify              => $sensu::check_notify,
```

Comment:
Why the change from $::sensu to $sensu?  I believe the fully scoped variable is the preferred method now.